webpack.config.js

```diff
@@ -1,173 +0,0 @@
-const path = require("path");
-const webpack = require("webpack");
-const HtmlWebpackPlugin = require("html-webpack-plugin");
-const MiniCssExtractPlugin = require("mini-css-extract-plugin");
-const { CleanWebpackPlugin } = require("clean-webpack-plugin");
-const CopyPlugin = require("copy-webpack-plugin");
-const WorkboxPlugin = require("workbox-webpack-plugin");
-const MomentLocalesPlugin = require("moment-locales-webpack-plugin");
-const OptimizeCssAssetsPlugin = require("optimize-css-assets-webpack-plugin");
-const safePostCssParser = require("postcss-safe-parser");
-
-const prodPlugins = (isDev) => {
-  if (isDev) {
-    return [];
-  }
-
-  return [
-    new WorkboxPlugin.InjectManifest({
-      swSrc: "./src/service-worker.js",
```

Comment:
@bodhish hmm, that is my bad. I did not think the `webpack` config has other logic not handled in CRA + Tailwind. Let me check this. Thanks

Reply:
@bodhish CRA (> 4) supports Workbox InjectManifet and URL routes pre-cache. I have a feeling we could simply utilise their `serviceWorker` from a fresh CRA and integrate into our project. If we can stay with CRA and get what we need, it would make sense, right? What do you think?

There will still be enough code change, so I would like extra eyes on this, thanks.

https://create-react-app.dev/docs/making-a-progressive-web-app/

Reply:
100%, let's stick to CRA